charts/timescaledb-multinode/README.md

```diff
@@ -108,6 +127,13 @@ our [TimescaleDB > Tutorial: Scaling out TimescaleDB](https://docs.timescale.com
 to create distributed hypertables and start using multinode TimescaleDB.
 
 ### Connecting from another pod
+If you are connecting from another pod you can disable external access completely by changing the service typpe from LoadBalancer to ClusterIP. Edit the following in values.yaml:
```

Comment:
`service type`

Reply:
this should fall under `## Connecting from outside Kubernetes`

Reply:
because there you say, in order of connecting from outside, 1.2.3, otherwise you can use `type: ClusterIP`
at any rate this is not related to `### Connecting from another pod` since every service type will have here the same instructions